src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/OffenderAssessmentRepository.java

```diff
@@ -1,11 +1,19 @@
 package uk.gov.justice.hmpps.prison.repository.jpa.repository;
 
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
 import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderAssessment;
-import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderBooking;
 
+import java.util.List;
 import java.util.Optional;
 
 public interface OffenderAssessmentRepository extends CrudRepository<OffenderAssessment, OffenderAssessment.Pk> {
     Optional<OffenderAssessment> findByBookingIdAndAssessmentSeq(Long bookingId, Integer assessmentSeq);
+
+    @Query("SELECT oa FROM OffenderAssessment oa " +
```

Comment:
Could this be done using the JPA finder methods or would it end up being loads more work?